src/schemas/json/agripparc-1.2.json

```diff
@@ -72,6 +72,6 @@
       "enum": ["https://json.schemastore.org/agripparc-1.2.json"]
     }
   },
-  "title": "JSON schema for the Agrippa config file",
+  "title": "json schema for the agrippa config file",
```

Comment:
Agrippa is a name, so shouldn't it still have a capital A?

Reply:
I guess it could. The reason to do this change is to conform our contributing guide. I've recently proposed change that all titles must have lower case values to be like more a programming language types visually. This is used when VS Code extension shown errors when u pass smth wrong instead of object. This PR just makes everything standardized. :) Maybe we can find a better way to write titles but my main point is be consistent.